### DISCUSSION TITLE
删除tablet以后，数据库还在寻找这个副本一直报错> 1105 - errCode = 2, detailMessage = Failed to get scan range, no queryable replica found in tablet: 49860592

### DISCUSSION THREAD

**Discussion by Ustinov-lwp** [apache/doris#22438] [A - General / Q&A]
这个tablet已经通过停止be，调用元数据管理工具进行删除，但还是在提醒这个错误，现在没任何解决思路了，求助官方和大神帮帮忙，数据库现在无法进行查询、删除等操作。

数据库版本号：1.2.6

**Comment by Ustinov-lwp** [comment]
现阶段已经过去一天，自动修复还是一个副本都没有修复成功，也找不到其他办法来解决，尝试过以下方式：
标记损坏的副本：
ADMIN SET REPLICA STATUS PROPERTIES("tablet_id" = "*****", "backend_id" = "*****", "status" = "bad");

填补空白副本：
ADMIN SET FRONTEND CONFIG ("recover_with_empty_tablet" = "true");

curl -X POST "http://10.0.0.200:8040/api/pad_rowset?tablet_id=49860592&start_version=313285&end_version=313285"

删除副本元数据：
curl -X POST "http://10.0.0.200:8040/api/restore_tablet?tablet_id=49860592\&schema_hash=1509357443"

注：
尝试重启、删除整个分区等操作，现数据库没有进行分组

**Reply by liuyi-stack** [reply]
> 现阶段已经过去一天，自动修复还是一个副本都没有修复成功，也找不到其他办法来解决，尝试过以下方式： 标记损坏的副本： ADMIN SET REPLICA STATUS PROPERTIES("tablet_id" = "_**", "backend_id" = "**_", "status" = "bad");
> 
> 填补空白副本： ADMIN SET FRONTEND CONFIG ("recover_with_empty_tablet" = "true");
> 
> curl -X POST "http://10.0.0.200:8040/api/pad_rowset?tablet_id=49860592&start_version=313285&end_version=313285"
> 
> 删除副本元数据： curl -X POST "[http://10.0.0.200:8040/api/restore_tablet?tablet_id=49860592\&schema_hash=1509357443](http://10.0.0.200:8040/api/restore_tablet?tablet_id=49860592%5C&schema_hash=1509357443)"
> 
> 注： 尝试重启、删除整个分区等操作，现数据库没有进行分组

请问有解决吗，我是断电后重启启动执行sql查询报这个错误的

**Reply by ixzc** [reply]
> 现阶段已经过去一天，自动修复还是一个副本都没有修复成功，也找不到其他办法来解决，尝试过以下方式： 标记损坏的副本： ADMIN SET REPLICA STATUS PROPERTIES("tablet_id" = "_**", "backend_id" = "**_", "status" = "bad");
> 
> 填补空白副本： ADMIN SET FRONTEND CONFIG ("recover_with_empty_tablet" = "true");
> 
> curl -X POST "http://10.0.0.200:8040/api/pad_rowset?tablet_id=49860592&start_version=313285&end_version=313285"
> 
> 删除副本元数据： curl -X POST "[http://10.0.0.200:8040/api/restore_tablet?tablet_id=49860592\&schema_hash=1509357443](http://10.0.0.200:8040/api/restore_tablet?tablet_id=49860592%5C&schema_hash=1509357443)"
> 
> 注： 尝试重启、删除整个分区等操作，现数据库没有进行分组

你这是几副本的？

**Reply by ixzc** [reply]
> > 现阶段已经过去一天，自动修复还是一个副本都没有修复成功，也找不到其他办法来解决，尝试过以下方式： 标记损坏的副本： ADMIN SET REPLICA STATUS PROPERTIES("tablet_id" = "_**", "backend_id" = "**_", "status" = "bad");
> > 填补空白副本： ADMIN SET FRONTEND CONFIG ("recover_with_empty_tablet" = "true");
> > curl -X POST "http://10.0.0.200:8040/api/pad_rowset?tablet_id=49860592&start_version=313285&end_version=313285"
> > 删除副本元数据： curl -X POST "[http://10.0.0.200:8040/api/restore_tablet?tablet_id=49860592&schema_hash=1509357443](http://10.0.0.200:8040/api/restore_tablet?tablet_id=49860592%5C&schema_hash=1509357443)"
> > 注： 尝试重启、删除整个分区等操作，现数据库没有进行分组
> 
> 请问有解决吗，我是断电后重启启动执行sql查询报这个错误的

断电重启导致的问题，你这个是几个副本的？

**Reply by liuyi-stack** [reply]
> > > 现阶段已经过去一天，自动修复还是一个副本都没有修复成功，也找不到其他办法来解决，尝试过以下方式： 标记损坏的副本： ADMIN SET REPLICA STATUS PROPERTIES("tablet_id" = "_**", "backend_id" = "**_", "status" = "bad");
> > > 填补空白副本： ADMIN SET FRONTEND CONFIG ("recover_with_empty_tablet" = "true");
> > > curl -X POST "http://10.0.0.200:8040/api/pad_rowset?tablet_id=49860592&start_version=313285&end_version=313285"
> > > 删除副本元数据： curl -X POST "[http://10.0.0.200:8040/api/restore_tablet?tablet_id=49860592&schema_hash=1509357443](http://10.0.0.200:8040/api/restore_tablet?tablet_id=49860592%5C&schema_hash=1509357443)"
> > > 注： 尝试重启、删除整个分区等操作，现数据库没有进行分组
> > 
> > 
> > 请问有解决吗，我是断电后重启启动执行sql查询报这个错误的
> 
> 断电重启导致的问题，你这个是几个副本的？

单副本 

PROPERTIES (
"replication_allocation" = "tag.location.default: 1",
"min_load_replica_num" = "-1",
"is_being_synced" = "false",
"storage_medium" = "hdd",
"storage_format" = "V2",
"inverted_index_storage_format" = "V1",
"enable_unique_key_merge_on_write" = "true",
"light_schema_change" = "true",
"disable_auto_compaction" = "false",
"enable_single_replica_compaction" = "false",
"group_commit_interval_ms" = "10000",
"group_commit_data_bytes" = "134217728",
"enable_mow_light_delete" = "false"
);;

**Reply by ixzc** [reply]
单副本的话，有副本损坏了，没有办法修复了，你可以看下这个文章哈
https://mp.weixin.qq.com/s/iJEGnnLMGuuv5KHy-skxLw

**Reply by liuyi-stack** [reply]
> 单副本的话，有副本损坏了，没有办法修复了，你可以看下这个文章哈 https://mp.weixin.qq.com/s/iJEGnnLMGuuv5KHy-skxLw
现在已经创建的表还能重新调整为三副本吗

**Comment by mizuki1412** [comment]
请问后来有解决嘛？
